src/v2/ion/i18nTransformer.js

```diff
@@ -100,13 +102,12 @@ const I18N_OVERRIDE_MAPPINGS = {
   'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
   'challenge-authenticator.okta_verify.credentials.totp': 'oie.okta_verify.totp.enterCodeText',
   'challenge-authenticator.google_authenticator.credentials.otp': 'oie.google_authenticator.otp.enterCodeText',
-
+  'challenge-authenticator.del_oath.credentials.passcode': 'oie.on_prem.verify.passcode.label',
   'enroll-profile.userProfile.lastName': 'oie.user.profile.lastname',
```

Comment:
nit: An empty line between `challenge-authenticator` and `enroll-profile`

Reply:
sorry! so many conflicts i must've deleted this